Fix ConceptLink warnings in virt/managing_vms (#111391)
openshift/openshift-docs: virt/managing_vms/advanced_vm_management/virt-managing-virtual-machines-by-using-openshift-gitops.adoc

.Prerequisites

> * You have a GitHub account. For instructions to set up an account, see Creating an account on GitHub.

Author: ```suggestion
* You have a GitHub account.
```

> * OpenShift Virtualuzation has been installed on your OpenShift cluster. For instructions, see OpenShift Virtualization installation.

Author: ```suggestion
* {VirtProductName} has been installed on your {product-title} cluster.
```

> * The OpenShift GitOps operator has been installed on your OpenShift cluster. For instructions, see Installing GitOps.

Author: ```suggestion
* The OpenShift GitOps Operator has been installed on your {product-title} cluster.
```

.Procedure

> Follow the _Manage OpenShift virtual machines with GitOps_ learning path in performing these steps:

Author: ```suggestion
Follow the "Manage OpenShift virtual machines with GitOps" learning path in performing these steps:
```

> Follow the _Manage OpenShift virtual machines with GitOps_ learning path in performing these steps:

Author: Outside scope of this PR

[role="_additional-resources"]

> .Additional resources

Author: ```suggestion
```

[role="_additional-resources"]
== Additional resources
* link:https://docs.github.com/en/get-started/start-your-journey/creating-an-account-on-github[Creating an account on GitHub]
* xref:../../../virt/install/installing-virt.adoc#installing-virt[OpenShift Virtualization installation]
* link:https://docs.openshift.com/gitops/1.15/installing_gitops/preparing-gitops-install.html[Installing GitOps]
* link:https://developers.redhat.com/learning/learn:manage-openshift-virtual-machines-gitops/resource/resources:connect-and-configure-external-repository-argo-cd-virtual-machines[the _Manage OpenShift virtual machines with GitOps_ learning path]

> * link:https://docs.openshift.com/gitops/[OpenShift GitOps documentation]

Author: Must move to the assembly
